app/assets/javascripts/components/story/CollapsedStory/index.js

```diff
@@ -41,49 +42,61 @@ export const Container = ({
   isHighlightable,
   isDragging,
   provided
-}) => (
-  <div
-    className={storyClassName(story, className, isDragging)}
-    onClick={onToggle}
-    title={title}
-    {...provided.draggableProps}
-    {...provided.dragHandleProps}
-    ref={provided.innerRef}
-  >
-    <StoryPopover story={story}>
-      <div className='Story__icons-block'>
-        <StoryIcon storyType={story.storyType} />
-        <CollapsedStoryEstimate estimate={story.estimate} />
-        <StoryDescriptionIcon description={story.description} />
-      </div>
-    </StoryPopover>
+}) => {
+  const isLoading = Boolean(story.loading);
```

Comment:
Why is this explicit coercion necessary?

Reply:
Truth be told, it wasn't necessary at all. Thanks!